app-misc/asciinema/asciinema-2.0.1.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="Terminal session recorder"
+DESCRIPTION="asciinema lets you easily record terminal sessions and replay \
+them in a terminal as well as in a web browser. It is the best companion of \
+asciinema.org."
+HOMEPAGE="https://asciinema.org/"
+COPYRIGHT="2011-2017 Marcin Kulik"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/asciinema/asciinema/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="7087b247dae36d04821197bc14ebd4248049592b299c9878d8953c025ac802e4"
+SOURCE_FILENAME="asciinema-$portVersion.tar.gz"
+PATCHES="asciinema-$portVersion.patchset"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	asciinema = $portVersion
+	cmd:asciinema
+	"
+REQUIRES="
+	haiku
+	cmd:python3
+	"
+REPLACES="
+	python3_asciinema
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	setuptools_python3
+	"
+BUILD_PREREQUIRES="
+	cmd:python3
+	cmd:sed
+	"
+
+TEST_REQUIRES="
+	nose_python3
+	"
+
+BUILD()
+{
+	rm -rf build
+	python3 setup.py build
+}
+
+INSTALL()
+{
+	sed -i -e "s|documentation/man|$relativeManDir|" \
```

Comment:
Suggestion, maybe:
```diff
-	sed -i -e "s|documentation/man|$relativeManDir|" \
+	sed -i \
+		-e "s|documentation/man|$relativeManDir|" \
		-e "s|documentation/packages/asciinema|$relativeDocDir|" \
		setup.py
```

Reply:
The misalignment was getting to me, too, if that's what this is for. Thanks for the review!